robot/web/stream/server.go

```diff
@@ -664,7 +672,11 @@ func (server *Server) refreshVideoSources(ctx context.Context) {
 			continue
 		}
 		existing, ok := server.videoSources[cam.Name().SDPTrackName()]
-		src := camerautils.VideoSourceFromCamera(ctx, cam)
+		src, err := camerautils.VideoSourceFromCamera(ctx, cam)
+		if err != nil {
+			server.logger.Errorf("error creating video source from camera: %v", err)
+			continue
+		}
```

Comment:
put this block before the `existing, ok` assignment as you don't use those variables until after this if check